src/core-sdk-tasks/core-sdk-tasks.csproj

```diff
@@ -18,6 +18,8 @@
     <PackageReference Include="System.Reflection.Metadata" Version="8.0.0" />
     <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" Condition="'$(DotNetBuildFromSource)' != 'true'" />
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
+    <PackageReference Include="System.Formats.Asn1" Version="$(SystemFormatsAsn1PackageVersion)"/>
+    <PackageReference Include="Microsoft.IO.Redist" Version="$(MicrosoftIORedistPackageVersion)" Condition="'$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
I'm not sure what this project is doing.  I see it referencing MSBuild and setting `CopyLocalLockFileAssemblies` to true.  Is this meant to produce a runnable layout of MSBuild?

Reply:
It's all of the tasks that installer uses to generate our zip/exe/pkg/nupkgs. I believe all the tasks are used in redist.csproj

Reply:
@ericstj checking in if you had more questions here. This csproj is just used for the internal layout build of the SDK product but never shipped. It's flagged in CG which is why the fix here.

Reply:
It looks to me like it's for buiilding some helper tasks that get used during the build of this repo.  My theory is that it shouldn't even be pulling down MSbuild and copying to it's output directory as I bet that's never even used.

You could get yourself out of the business of updating these packages if you referenced MSBuild in a different way.